.github/workflows/pr.yml

```diff
@@ -43,5 +44,6 @@ jobs:
       - run: ls -lah ./pkg/apt-spy2*
       - run: gem install pkg/apt-spy2*
       - run: apt-spy2 check --strict
+      - run: sudo env "PATH=$PATH" apt-spy2 fix --commit --strict
```

Comment:
```suggestion
      - run: sudo env "PATH=$PATH" apt-spy2 fix --commit --strict
      - run: apt-get update
```

Reply:
@till What are we testing with adding this step here?

Reply:
If update still works as expected. This would have caught the frozen string stuff. Not ideal, but it works?

Reply:
The `commit` command would've caught the frozen string error, while the update wouldn't have! Although, with `update`, we'd know that mirror picked during CI works, which will be a more thorough test but is a non-deterministic  one, so not really sure if it adds any value. What do you think?